keyboards/meetlab/kalice/keymaps/default/keymap.c

```diff
@@ -0,0 +1,35 @@
+/* Copyright 2022 LXF-YZP(yuezp)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+[0] = LAYOUT(
+   KC_ESC,  KC_GRV,  KC_1,    KC_2,    KC_3,   KC_4,    KC_5,    KC_6,    KC_7,    KC_8,     KC_9,     KC_0,    KC_MINS, KC_EQL,  KC_BSPC,  KC_INS, 
+   KC_PGUP, KC_TAB,  KC_Q,    KC_W,    KC_E,   KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,     KC_O,     KC_P,    KC_LBRC, KC_RBRC, KC_BSLS,  KC_DEL, 
+   KC_PGDN, KC_CAPS, KC_A,    KC_S,    KC_D,   KC_F,    KC_G,    KC_H,    KC_J,    KC_K,     KC_L,     KC_SCLN, KC_QUOT,          KC_ENT,   MO(1), 
+		    KC_LSFT, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_B,    KC_N,    KC_M,     KC_COMM,  KC_DOT,  KC_SLSH, KC_RSFT, KC_UP, 
+		    KC_LCTL, KC_LGUI, KC_LALT,         KC_SPC,                    KC_SPC,  KC_RCTL,  KC_RALT,                    KC_LEFT, KC_DOWN,  KC_RGHT),
```

Comment:
```suggestion
    KC_LSFT, KC_Z,    KC_X,    KC_C,   KC_V,    KC_B,    KC_B,    KC_N,    KC_M,     KC_COMM,  KC_DOT,  KC_SLSH, KC_RSFT, KC_UP, 
    KC_LCTL, KC_LGUI, KC_LALT,         KC_SPC,                    KC_SPC,  KC_RCTL,  KC_RALT,                    KC_LEFT, KC_DOWN,  KC_RGHT),
```

Reply:
I'm sorry for the trouble caused by the format problem